.github/workflows/lint.yml

```diff
@@ -24,13 +23,13 @@ jobs:
       - name: Check linter configuration
         run: make lint-config
       - name: Run linter
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@v7
         with:
-          version: v1.63.4
+          version: 2.0.2
 
   yamllint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Run yamllint make target
-        run: make yamllint
+        run: make yamllint
```

Comment:
NIT: Could you please leave an empty line at the end of this file ( see the symble red, it should not appears )